reana_job_controller/rest.py

```diff
@@ -215,7 +219,13 @@ def create_job():  # noqa
         return jsonify({"job": msg}), 500
     with current_app.app_context():
         job_manager_cls = current_app.config["COMPUTE_BACKENDS"][compute_backend]()
-        job_obj = job_manager_cls(**job_request)
+        try:
+            job_obj = job_manager_cls(**job_request)
+        except (
+            REANAKubernetesWrongMemoryFormat,
+            REANAKubernetesMemoryLimitExceeded,
+        ) as e:
+            return jsonify({"message": e.message}), 500
```

Comment:
Minor: Perhaps a `403` error would be more representative for `REANAKubernetesMemoryLimitExceeded ` and `400` for the `REANAKubernetesWrongMemoryFormat `.